airplanedb.py

```diff
@@ -177,4 +177,290 @@ def reset_db(self):
+
+        ''' insert test flights '''
+        insert_flight_1 = """ INSERT INTO FLIGHT (AC_ID, F_DISTANCE,
+                            F_DEPARTURETIME, F_ARRIVALTIME, F_DEPARTUREAIRPORTID, F_ARRIVALAIRPORTID,
+                            F_DEPARTUREGATEID, F_ARRIVALGATEID, F_STATUS)
+                            VALUES (1, 5000, '01-10-2018:23:23', '01-12-2018:06:23', 'YVR', 'TPE', 'A1', 'E5', 'PEDNGING')
+                            """
+        insert_flight_2 = """ INSERT INTO FLIGHT (AC_ID, F_DISTANCE,
+                            F_DEPARTURETIME, F_ARRIVALTIME, F_DEPARTUREAIRPORTID, F_ARRIVALAIRPORTID,
+                            F_DEPARTUREGATEID, F_ARRIVALGATEID, F_STATUS)
+                            VALUES (3, 250, '01-08-2018:05:23', '01-08-2018:06:23', 'LAX', 'YVR', 'C5', 'B3', 'PEDNGING')
+                            """
+
+        insert_flight_3 = """ INSERT INTO FLIGHT (AC_ID, F_DISTANCE,
+                            F_DEPARTURETIME, F_ARRIVALTIME, F_DEPARTUREAIRPORTID, F_ARRIVALAIRPORTID,
+                            F_DEPARTUREGATEID, F_ARRIVALGATEID, F_STATUS)
+                            VALUES (2, 4500, '01-13-2018:05:23', '01-13-2018:17:23', 'TPE', 'JFK', 'E2', 'A3', 'PEDNGING')
+                            """
+
+        try:
+            cursor.execute(insert_flight_1)
+            print(('Created new {0} from {1} to {2}').format('FLIGHT', 'YVR', 'TPE'))
+            cursor.execute(insert_flight_2)
+            print(('Created new {0} from {1} to {2}').format('FLIGHT', 'LAX', 'YVR'))
+            cursor.execute(insert_flight_3)
+            print(('Created new {0} from {1} to {2}').format('FLIGHT', 'TPE', 'JFK'))
+            db.commit()
+        except Exception as e:
+            print(e)
+
+        ''' insert test employees '''
+        insert_employee_1 = """ INSERT INTO EMPLOYEE (E_HOURS, E_TYPE, E_WAGE)
+                            VALUES (70, "CAPTAIN", 53)
+                            """
+
+        insert_employee_2 = """ INSERT INTO EMPLOYEE (E_HOURS, E_TYPE, E_WAGE)
+                            VALUES (90.5, "ATTENDANT", 35)
+                            """
+
+        insert_employee_3 = """ INSERT INTO EMPLOYEE (E_HOURS, E_TYPE, E_WAGE)
+                            VALUES (15, "ATTENDANT", 32)
+                            """
+        try:
+            cursor.execute(insert_employee_1)
+            print(('Created new {0}: EID = {1}').format('EMPLOYEE', 1))
+            cursor.execute(insert_employee_2)
+            print(('Created new {0}: EID = {1}').format('EMPLOYEE', 2))
+            cursor.execute(insert_employee_3)
+            print(('Created new {0}: EID = {1}').format('EMPLOYEE', 3))
+            db.commit()
+        except Exception as e:
+            print(e)
+
+        ''' insert test itinerary '''
+        insert_itinerary_1 = """ INSERT INTO ITINERARY(I_ID, I_SEATTYPE, I_SEATCOST, I_STATUS, C_ID)
+                             VALUES ('XMASXPECIAL', 'FIRSTCLASS', 153.2, 'PENDING', 1)
```

Comment:
should we be specifying the primary key of `ITINERARY` when inserting a new itinerary? also, I thought this was supposed to reference `FLIGHT` as well